frontend/src/components/ResourceList/ResourceList.js

```diff
@@ -1,14 +1,16 @@
 import { nanoid } from "nanoid";
-
+import ErrorFetchingResources from "../../pages/ErrorFetchingResources/ErrorFetchingResources";
 import Resource from "../Resource/Resource";
 
 export default function ResourceList({ resources }) {
   return (
     <div className="resource-list">
-      {resources &&
-        resources.map((resource) => (
-          <Resource resource={resource} key={nanoid()} />
-        ))}
+      {resources ?( 
+        resources &&
+          resources.map((resource) => (
+            <Resource resource={resource} key={nanoid()} />
+          ))
+      ):(<ErrorFetchingResources/>) }
```

Comment:
I'm not sure this is also need in here. Is it not enough to only do it in the `Resource.js` file? I checked and it doesn't make a difference because it's resolved at a higher level/component

If not needed don't forget to remove unused variables/imports